services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs

```diff
@@ -177,19 +177,21 @@ impl SentinelS2L2aCogsDataProvider {
                         },
                         symbology: Some(Symbology::Raster(RasterSymbology {
                             opacity: 1.0,
-                            colorizer: Colorizer::linear_gradient(
-                                vec![
-                                    (0.0, RgbaColor::white())
-                                        .try_into()
-                                        .expect("valid breakpoint"),
-                                    (10_000.0, RgbaColor::black())
-                                        .try_into()
-                                        .expect("valid breakpoint"),
-                                ],
-                                RgbaColor::transparent(),
-                                RgbaColor::transparent(),
-                            )
-                            .expect("valid colorizer"),
+                            colorizer:
+                                geoengine_datatypes::operations::image::Colorizer::linear_gradient(
```

Comment:
Why do we need the full path here?

Reply:
The `linear_gradient` method is only available on the Colorizer of the datatypes crate. But for including it into the API I have to use the services crate counterpart. To make it apparent that this Colorizer is not the one used in the API, but only converted `into` the actual Colorizer, I put the full path here.